snippets/use_active_record_false.rb

```diff
@@ -0,0 +1,39 @@
+require "bundler/inline"
+
+gemfile(true) do
+  source "https://rubygems.org"
+
+  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+  # Those Gemfiles carefully pick the right versions depending on
+  # settings in the ENV, `.rails-version` and `maintenance-branch`.
+  eval_gemfile './Gemfile-sqlite-dependencies'
+  eval_gemfile './Gemfile-rspec-dependencies'
+  eval_gemfile './Gemfile-rails-dependencies'
+
+  gem "ammeter"
+  gem "rspec-rails", path: "./"
+end
+
+# Initialization
+require "active_record/railtie"
+require "rspec/rails"
+
+# Run specs on exit
+require "rspec/autorun"
+
+# RSpec configuration
+RSpec.configure do |config|
+  config.use_active_record = false
+end
+
+# Rails project code
+class Command
```

Comment:
I'd rather we did't use `Command` purely because this does happen with any name and `Command` sounds special, which its not

Reply:
Sure, I'll rename it. I believe it was coming from the original reproduction snippet.